.github/workflows/clean.yml

```
@@ -6,12 +6,15 @@ on:
 
 jobs:
   scan-old-package:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
 
     # disable for now
     if: false
 
     steps:
+      - name: Setup permssions monitoring
```

```suggestion
      - name: Setup permissions monitoring
```
And few other places